tests/commons/j2/test_j2_functions.py

```
@@ -11,290 +12,250 @@
+
+
+def test_rigid_bool() -> None:
+    # None conversion required before formatting
+    with pytest.raises(AssertionError):
+        xsd_fmt(None, "xsd:boolean")
+        assert False, "None should not be accepted as bool value"
+
+    # [] conversion (empty or not) required before formatting
+    with pytest.raises(AssertionError):
+        xsd_fmt([], "xsd:boolean")
+        assert False, "[] should not be accepted as bool value"
```

Ruff (0.8.2)

`pytest.raises()` block should contain a single simple statement

(PT012)